### PR TITLE
Request for TSE Review Before Final Submission: Design Teardown Task

### PR DESCRIPTION
**Pull request by Calvinoea**
Hello,

We are making a pull request for TSE review of our Design Teardown task before final submission. 

The project has been carried out using HTML, CSS, Stickler, and HTML validation. The project objective was to create a heatmap that highlights areas on the https://www.smashingmagazine.com/ page, according to their hierarchy and visibility. Gray, light gray, and dark gray colours were used to identify the different areas of the page in the heatmap. 

Please find below links to the live demo : 

https://gracious-jang-d7edc2.netlify.com/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,68 @@
...
+- HTML
+- CSS
+
+  
+## Live Project  
+
+[Design Teardown](https://gracious-jang-d7edc2.netlify.com/)  
+
+https://rawcdn.githack.com/Calvinoea/Design-Teardown-Smashing-Magazine/tree/feature/index.html
+https://raw.githack.com/Calvinoea/Design-Teardown-Smashing-Magazine/tree/feature/index.html
```
Comment (euqueme) [inline README.md]: the links to the Live project aren't working please delete them from your Readme or replace them with working links

File: index.html [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+    <meta name="description" content="description here">
+    <meta name="keywords" content="keywords,here">
+    <script src="https://kit.fontawesome.com/dcc95333f3.js" crossorigin="anonymous"></script>
+    <link rel = "stylesheet"  type = "text/css" href = "style.css"/>
+    <link href='http://fonts.googleapis.com/css?family=Open+Sans' rel='stylesheet' type='text/css'>
+    </head>
+    <body>
+
+        <header>
+                <!-- Navigation-->
```
Comment (euqueme) [inline index.html]: delete this comment on line #16

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Hello @Calvinoea and @Mupa1  :lollipop: :lollipop: 

### Good job so far :medal_sports:  but there are some things left to do still :handshake: 

## Playing with Typography

- [ ] Add meaningless text to the articles that have text in the smashing magazine and play with the fonts properties (Download a couple of fonts and import them in your HTML or CSS file, and of course use them :dancer: ), remember to set the background colors to white for these articles since the fonts are the ones who have to catch the user's attention. just like is stated in the Odin project:

image


## Footer

- [ ]  in addition, please also add the centered links in the footer

## Additional Microverse Requirements (Pathwright)

- [ ] Linters config: your linters aren't working perhaps you need to activate the project in [stickler](https://stickler-ci.com/)

- [ ] Check HTML markup with this validator https://validator.w3.org/#validate_by_upload

I have spotted some hiccups in your code please address them in the comments below :arrow_down_small: 

[Optional] I deeply encourage you to use the [`<nav>` tag](http://html5doctor.com/nav-element/) for the major navigation links.

### Thats it team! :raised_hands: 

### Please :pray:  do the changes :memo:  and submit back for a code review when you are done :rocket: 

any question? reach to me on slack **@ Maru**

Happy Coding! :woman_technologist: 

PROJECT STATUS: REQUESTED CHANGES :traffic_light:

**Comment by Calvinoea** [comment]
> > ### Pull Request under the review by Maria Eugenia Queme P. (TSE)
> 
> ### Hello @Calvinoea and @Mupa1 🍭 🍭
> ### Good job so far 🏅 but there are some things left to do still 🤝
> ## Playing with Typography
> * [ ]  Add meaningless text to the articles that have text in the smashing magazine and play with the fonts properties (Download a couple of fonts and import them in your HTML or CSS file, and of course use them 💃 ), remember to set the background colors to white for these articles since the fonts are the ones who have to catch the user's attention. just like is stated in the Odin project:
> 
> image
> 
> ## Footer
> * [ ]   in addition, please also add the centered links in the footer
> 
> ## Additional Microverse Requirements (Pathwright)
> * [ ]  Linters config: your linters aren't working perhaps you need to activate the project in [stickler](https://stickler-ci.com/)
> * [ ]  Check HTML markup with this validator https://validator.w3.org/#validate_by_upload
> 
> I have spotted some hiccups in your code please address them in the comments below 🔽
> 
> [Optional] I deeply encourage you to use the [`<nav>` tag](http://html5doctor.com/nav-element/) for the major navigation links.
> 
> ### Thats it team! 🙌
> ### Please 🙏 do the changes 📝 and submit back for a code review when you are done 🚀
> any question? reach to me on slack **@ Maru**
> 
> Happy Coding! 👩‍💻
> 
> PROJECT STATUS: REQUESTED CHANGES 🚥

Hello @euqueme . Thank you for your feedback. We will make the changes as discussed.

**Review by nikasulo — APPROVED** [review]
# Great job @Calvinoea and @Mupa1!
All changes seem to have been made correctly. Please merge and keep moving forward! :100: Also, nice work considering the optional change suggested by the previous TSE. This is really solid work. Please keep up your hard work throughout the curriculum and I wish you all the best! :rocket: See you on top :medal_military: 

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

**Comment by Calvinoea** [comment]
> # Great job @Calvinoea and @Mupa1!
> All changes seem to have been made correctly. Please merge and keep moving forward! 💯 Also, nice work considering the optional change suggested by the previous TSE. This is really solid work. Please keep up your hard work throughout the curriculum and I wish you all the best! 🚀 See you on top 🎖
> 
> ##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

Thank you, @Oluwadamilareolusakin . We appreciate your feedback. The merge has been made.

**Pull request merged**